x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/XPackSettings.java

```diff
@@ -121,6 +121,12 @@ private XPackSettings() {
     /** Setting for enabling or disabling vectors. Defaults to true. */
     public static final Setting<Boolean> VECTORS_ENABLED = Setting.boolSetting("xpack.vectors.enabled", true, Setting.Property.NodeScope);
 
+    /**
+     * Setting for enabling or disabling geo features. Defaults to true.
+     */
+    public static final Setting<Boolean> GEO_ENABLED = Setting.boolSetting("xpack.geo.enabled", true,
```

Comment:
should we have this setting? Do we foresee anyone wanting to disable any of this?